specs/configure_monitoring.adoc

```diff
@@ -162,17 +290,11 @@ https://github.com/Tendrl/specifications/pull/8/files
 
 == Testing:
 
-This adds 2 ways of overriding the tendrl chosen default monitoring
-configurations as described under "Other end user impact" and
-"Tendrl API impact" which need to be tested.
+This adds apis as described in "Tendrl API impact" which need to be tested.
 
 == Documentation impact:
 
-The items to be documented includes an api exposed for overriding the default
-monitoring configuration and also a config file that can be used to have custom
-default values that override the tendrl chosen defaults as desscribed under:
-"Other end user impact" and the api is described in "Tendrl API impact"
-
+This adds apis as described in "Tendrl API impact" which need to be documented
 
 == References:
```

Comment:
will the tendrl-api related changes captured ion a different spec? I think it shouldnot. You need to raise github issues to track those and add the links here